phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ApplicationSideBar.java

```diff
@@ -158,7 +158,7 @@ private ToggleButton createAllCategoriesToggleButton() {
     private ToggleButton createCategoryToggleButton(CategoryDTO category) {
         final LeftToggleButton categoryButton = new LeftToggleButton(category.getName());
 
-        categoryButton.setId(String.format("%sButton", category.getName().toLowerCase()));
+        categoryButton.setId(String.format("%sButton", category.getId().toLowerCase()));
```

Comment:
It's really not important. The thing is that I cannot use the `name` anymore because it can be different depending on the language. Apart from that there are no restrictions.